File: android/versioned-abis/expoview-abi40_0_0/src/main/java/abi40_0_0/expo/modules/speech/SpeechPackage.java
package abi40_0_0.expo.modules.speech;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import abi40_0_0.org.unimodules.core.BasePackage;
import abi40_0_0.org.unimodules.core.ExportedModule;

public class SpeechPackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context reactContext) {
    return Collections.singletonList((ExportedModule) new SpeechModule(reactContext));
  }
}

File: android/versioned-abis/expoview-abi48_0_0/src/main/java/abi48_0_0/expo/modules/speech/SpeechPackage.kt
package abi48_0_0.expo.modules.speech

import android.content.Context
import abi48_0_0.expo.modules.core.BasePackage

class SpeechPackage : BasePackage() {
  override fun createExportedModules(context: Context) = listOf(SpeechModule(context))
}
